include/net/socket.h

```diff
@@ -821,6 +822,12 @@ static inline char *inet_ntop(sa_family_t family, const void *src, char *dst,
 /** sockopt: disable IPv6 ICMP replies */
 #define SO_IPV6_ECHO_REPLY 32
 
+/**
+ * sockopt: Receive timeout
+ * Applies to receive functions like recv(), but not to connect()
+ */
+#define SO_RCVTIMEO 20
```

Comment:
This "macro" is already defined a few lines above which is what is causing the documentation build issue.

Reply:
I see, the first one is from "[nrf noup] include: net: add NCS extensions" the second from upstream. Thanks @utzig

Reply:
That would require one revert further than before, so I went one more to the last [nrf temphack].